mwapi/session.py

```diff
@@ -84,66 +92,42 @@ def logout(self):
         Logs out of the session with MediaWiki
         """
         self.post(action='logout')
-        self.is_authenticated = False
```

Comment:
So this is basically stripping away password auth, and will add support for mwoauth in some way in the future. Is that correct?